dev

```diff
@@ -3,7 +3,7 @@
 set -euo pipefail
 
 # Bump this counter to force rebuilding `dev` on all machines.
-DEV_VERSION=41
+DEV_VERSION=42
```

Comment:
42 is the current version, you probably want to rebase and bump this again.

Reply:
Done